rover-apps/pdb/src/PDB_Module.cpp

```diff
@@ -0,0 +1,114 @@
+#include "PDB_Module.h"
+
+#include <cmath>
+
+#include "Logger.h"
+#include "PDB_config.h"
+#include "hw_bridge.h"
+
+/* Pins configuration for Load Monitoring */
+DigitalOut load1_5V_diag_en(LOAD1_5V_DIAG_EN);
+DigitalIn load1_5V_fault_n(LOAD1_5V_FAULT);
+
+DigitalOut load2_5V_diag_en(LOAD2_5V_DIAG_EN);
+DigitalIn load2_5V_fault_n(LOAD2_5V_FAULT);
+
+DigitalOut load3_5V_diag_en(LOAD3_5V_DIAG_EN);
+DigitalIn load3_5V_fault_n(LOAD3_5V_FAULT);
+
+DigitalOut load4_5V_diag_en(LOAD4_5V_DIAG_EN);
+DigitalIn load4_5V_fault_n(LOAD4_5V_FAULT);
+
+DigitalOut load5_5V_diag_en(LOAD5_5V_DIAG_EN);
+DigitalIn load5_5V_fault_n(LOAD5_5V_FAULT);
+
+DigitalOut load_17V_diag_en(LOAD_17V_DIAG_EN);
+DigitalIn load_17V_fault_n(LOAD_17V_FAULT);
+
+/* Pins configuration for Rail Monitoring */
+AnalogIn railBattery(RAIL_BATTERY_ANLG_IN);
+AnalogIn rail5V(RAIL_5V_ANLG_IN);
+AnalogIn rail17V(RAIL_17V_ANLG_IN);
+AnalogIn rail24V(RAIL_24V_ANLG_IN);
+DigitalIn rail24V_pgood_n(RAIL_24V_PGOOD_N);
+
+/* Pins configuration for Temperature Monitoring */
+AnalogIn temperatureADC(TEMPERATURE_ADC_IN);
```

Comment:
Let's make these private member vars of the PDB module class

Reply:
These are actually member functions of DigitalIn, DigitalOut, AnalogIn, and AnalogOut so I couldn't make them private variables of the PDB class.